jkube-kit/config/service/src/main/java/org/eclipse/jkube/kit/config/service/ApplyService.java

```diff
@@ -1390,4 +1394,55 @@ public void processCustomEntities(List<String> customResourceDefinitions, File r
             }
         }
     }
+
+    public static void applyNamespaceOrProjectIfPresent(ApplyService applyService, Collection<HasMetadata> entities) {
```

Comment:
Same as the others. Static method which accepts an instance of the enclosing class.
>> Remove static modifier and ApplyService instance parameter